scene/main/node.cpp

```diff
@@ -217,8 +217,13 @@ void Node::_notification(int p_notification) {
 void Node::_propagate_ready() {
 	data.ready_notified = true;
 	data.blocked++;
+
+	List<Node *> li;
```

Comment:
```suggestion
	TightLocalVector<Node *> li;
	li.reserve(data.children.size());
```
`List` is a linked list, it is almost always better to use a continious container instead.

Reply:
Thanks for the improvement. I will try to remember this method.